src/index.html

```diff
@@ -4,10 +4,229 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" href="./images/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page">
+    <header class="page__header header" id="home">
+      <div class="header__container">
+        <nav class="nav">
+          <input
+            id="menu-toggler"
+            class="nav__toggler"
+            type="checkbox"
+          >
+          <label
+            for="menu-toggler"
+            class="nav__togglerLabel"
+          >
+          </label>
+          <div class="nav__menu">
+            <img
+              class="logo nav__logo"
+              src="./images/Union.png"
+              alt="Logo"
+            >
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#home">
+                  Home
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">
+                  About us
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#gallery">
+                  Gallery
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#consultation">
+                  Consultation
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#testimonials">
+                  Testimonials
+                </a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">
+                  Contacts
+                </a>
+              </li>
+            </ul>
+            <p class="nav__phone">
+              +1 234 555-55-55
+            </p>
+            <a class="nav__link nav__link--phone" href="tel: +12345555555">
+              Call to order
+            </a>
+          </div>
+        </nav>
+        <img
+          class="header__logo logo"
+          src="./images/Union.png"
+          alt="Logo"
+        >
+        <a
+          class="header__linkTel"
+          href="tel:+12345555555"
+        >
+        </a>
+      </div>
+      <h1 class="header__title">Miami<br>Condo Kings</h1>
+    </header>
+    <main class="page__content">
+      <section
+        class="aboutUs"
```

Comment:
```suggestion
        class="about-us"
```
Better to use kebab-case for class names